components/src/components/molecules/PageButtons/PageButtons.tsx

```diff
@@ -40,6 +40,16 @@ const PageButton = styled.button<{ $selected?: boolean; $size: Size }>(
     background-color: ${theme.colors.background};
     transition: all 0.15s ease-in-out;
     cursor: pointer;
+    font-size: ${theme.fontSizes.body};
+    line-height: ${theme.lineHeights.body};
+    font-weight: ${theme.fontWeights.bold};
```

Comment:
should be medium weight

Reply:
It's been changed to bold in figma > thorin > pagination

Reply:
just realised it's not the weight but the color, should be `greyPrimary` (from figma)

Reply:
Ahh... yeah... fixed